### PR TITLE
T&A Bugfixes

### PR DESCRIPTION
**Pull request by fhelfer**
Found some interfering Errors while clicking through Test

### CODE REVIEW

File: Modules/Test/classes/class.ilTestRandomQuestionSetConfig.php [hunk 1]
```diff
@@ -384,7 +384,7 @@ public function removeQuestionSetRelatedData(): void
 
     public function resetQuestionSetRelatedTestSettings()
     {
-        $this->testOBJ->setResultFilterTaxIds(array());
```
Comment (fhelfer) [inline Modules/Test/classes/class.ilTestRandomQuestionSetConfig.php:387]: need some expertise with this

Comment (dsstrassner) [reply Modules/Test/classes/class.ilTestRandomQuestionSetConfig.php:387]: Hi @mbecker-databay, 

could you please give @fhelfer a hand?

Comment (mbecker-databay) [reply Modules/Test/classes/class.ilTestRandomQuestionSetConfig.php:387]: I think I fixed this particular one yesterday... 

        $this->testOBJ->getScoreSettingsRepository()->store(
            $this->testOBJ->getScoreSettings()->withResultDetailsSettings(
                $this->testOBJ->getScoreSettings()->getResultDetailsSettings()->withTaxonomyFilterIds(array())
            )
        );

**Comment by kergomard** [comment]
Thank you @fhelfer : I close this, because both issues have been fixed in the meantime.

**Pull request closed without merging**